nano/node/transport/udp.hpp

```diff
@@ -213,7 +269,7 @@ namespace transport
 		attempts;
 		std::unordered_map<nano::endpoint, syn_cookie_info> syn_cookies;
 		std::unordered_map<boost::asio::ip::address, unsigned> syn_cookies_per_ip;
-		nano::node & node;
+		std::unordered_map<nano::tcp_endpoint, std::vector<nano::endpoint>> response_channels;
```

Comment:
I think nano::network is responsible for managing response channels. Right now it's only udp_channels though we could create a new class tcp_channels which would also be inside nano::network and nano::network is responsible for live messages across all channel types.

nano::network should be able to handle the traffic in a channel-agnostic way.